### ISSUE TITLE
cannot remote debug

### ISSUE THREAD

**Issue by chrisdias** [Microsoft/vscode-azureappservice#789]
Try to remote debug...

```
Action: appService.StartRemoteDebug
Error type: Error
Error Message: Unable to establish connection to application: WebApp is started, but port is unreachable

Version: 0.10.0
OS: darwin
```

Repro steps:
* deploy a simple express/node app to app services on a B1 plan
* set a breapoint in `routes/index.js`
* right click on site and choose `Start Remote Debugging`

Results in the Output pane for App Services:

```
Fetching site configuration...
Checking app settings...
Updating site configuration to set remote debugging...
Updating site configuration done...
Starting tunnel proxy...
[WebApp Tunnel] Checking status, body: {"port":2222,"state":"STARTED","canReachPort":true,"msg":""}
[WebApp Tunnel] Checking status, body: {"port":49494,"state":"STARTING","canReachPort":false,"msg":"Unable to connect to WebApp"}
[WebApp Tunnel] Checking status, body: {"port":49494,"state":"STARTING","canReachPort":false,"msg":"Unable to connect to WebApp"}
[WebApp Tunnel] Checking status, body: {"port":49494,"state":"STARTING","canReachPort":false,"msg":"Unable to connect to WebApp"}
[WebApp Tunnel] Checking status, body: {"port":49494,"state":"STARTED","canReachPort":false,"msg":"Unable to connect to WebApp"}
Error: Unable to establish connection to application: WebApp is started, but port is unreachable
```

**Comment by fiveisprime** [comment]
This is a limitation of overriding `npm start` on older node images and a fix is being tested in `node@10.1.0`.

Since you're using the default express generator scaffolded app, you'll need to be on node runtime version 10.1.0 or override your debug start command so that app service can correctly initialize the app for debugging.

Try this again on 10.1.0 and verify that you're able to connect.

**Comment by fiveisprime** [comment]
I'm going to close this since it's a known limitation of the existing images. See #774 for the updated start config shipping more broadly.
